cohere/responses/generation.py

```diff
@@ -29,13 +29,15 @@ def __init__(
         likelihood: float,
         token_likelihoods: List[TokenLikelihood],
         prompt: str = None,
+        finish_reason: str = None,
```

Comment:
can we document (either in client.generate or in a new docstring here) what the possible values are for finish_reason?